src/cmd/build.js

```diff
@@ -0,0 +1,64 @@
+/* @flow */
+import path from 'path';
+import {createWriteStream} from 'fs';
+import streamToPromise from 'stream-to-promise';
+
+import * as fs from '../util/promised-fs';
+import {zipDir} from '../util/zip-dir';
+import {ProgramOptions} from '../program';
+import getValidatedManifest from '../util/manifest';
+
```

Comment:
Yeah, good idea to control this. I will set the max to two 2 because I am used to reading code in the [Python pep8 style](https://www.python.org/dev/peps/pep-0008/#blank-lines) of two blank lines between functions. I don't see an eslint way to enforce that rule exactly but it's ok :)

The original [setuptools](https://pypi.python.org/pypi/setuptools) code was the most infamous Python offender where there were entire pages of blank lines. It was refused entry into the standard library because of this.